From 75d1651c5c039b9fddf82c50272ccf83cb40250b Mon Sep 17 00:00:00 2001
From: Alec Larson <1925840+aleclarson@users.noreply.github.com>
Date: Tue, 26 Jul 2022 17:15:16 -0400
Subject: [PATCH] fix: use `px` for letter-spacing

---
 src/rules/typography.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rules/typography.ts b/src/rules/typography.ts
index 9db73b0..c1d5d03 100644
--- a/src/rules/typography.ts
+++ b/src/rules/typography.ts
@@ -88,7 +88,7 @@ export const fonts: Rule<Theme>[] = [
     /^(?:font-)?tracking-(.+)$/,
     ([, s], { theme }) => ({
       'letter-spacing':
-        theme.letterSpacing?.[s] || h.bracket.cssvar.global.rem(s),
+        theme.letterSpacing?.[s] || h.bracket.cssvar.global.px(s),
     }),
     { autocomplete: 'tracking-$letterSpacing' },
   ],
-- 
2.36.1

