From 3fe6d5fab3a921f76a27c2da75846d9ab688e7d1 Mon Sep 17 00:00:00 2001
From: Alec Larson <1925840+aleclarson@users.noreply.github.com>
Date: Tue, 26 Jul 2022 19:13:21 -0400
Subject: [PATCH] feat: set `outline-style: solid` along with outline width

---
 src/rules/behaviors.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/rules/behaviors.ts b/src/rules/behaviors.ts
index 85c07e2..8e8f833 100644
--- a/src/rules/behaviors.ts
+++ b/src/rules/behaviors.ts
@@ -7,6 +7,7 @@ export const outline: Rule<Theme>[] = [
   [
     /^outline-(?:width-|size-)?(.+)$/,
     ([, d], { theme }) => ({
+      'outline-style': 'solid',
       'outline-width': theme.lineWidth?.[d] ?? h.bracket.cssvar.global.px(d),
     }),
     { autocomplete: 'outline-(width|size)-<num>' },
-- 
2.36.1

